pages/home.jsx

```diff
@@ -14,7 +13,7 @@ var IconButton = require('../components/icon-button.jsx');
 
 var config = require('../lib/config');
 
-var blogFeedLoader = require('../lib/blog-feed-loader');
+var loadBlogPosts = require('../lib/blog-feed-loader');
```

Comment:
Since functions are like verbs, I changed the name of the function from `blogFeedLoader` (a noun) to `loadBlogPosts`...